### PR TITLE
Survey scanner

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
Survey Scanners - Drafted.

Adds survey scanners to the game. Survey Scanners are a replacement system for the survey handheld. They require going outside (horrific) and engaging with the game to find survey points, which can then be scanned for research points. As a side result they adjust dissection values to make it less useful, and can also be used to discover ore veins (which now start semi-hidden, as they are underground)

I need to touch up the code before it's ready for a merge but they're perfectly functional right now

- [ ]  have a design discussion to see if these should be used 
- [ ] if the answer is yes redesign and implement.

## Changelog

:cl:
add: Survey Scanners! A new-ish system for doing survey gameplay. That is to say there is actually gameplay now.
del: Survey handhelds
balance: Ore veins are now hidden until scanned properly.
/:cl:

### CODE REVIEW

**Review by thgvr — CHANGES_REQUESTED** [review]
I don't really think the ore vein change is that great. We already greatly reduced the occurrence of ore veins in rock walls, as drill mining and defense is a more reliable method to acquire large amounts of ore.

For the survey scanner, at this point we've all but phased out RND equipment on ships in favor of cargo, so I unfortunately can't really see this serving much of a purpose as is.

**Comment by Erikafox** [comment]
I was thinking about making it equipment for a set of outpost missions, but that'd require some actual design discussion.

File: code/modules/surgery/experimental_dissection.dm [hunk 1]
```diff
@@ -58,29 +58,33 @@
...
 		if(H?.dna?.species)
 			if(isabductor(H))
-				cost = (MAX_DISSECTION_REWARD*24)
+				cost = (BASE_HUMAN_REWARD*3)
 			else if(iszombie(H) || isshadow(H) || isandroid(H))
-				cost = (MAX_DISSECTION_REWARD*20)
-	else for(var/type in mob_mult_list) // THIS. ELSE IF(). ENDS. HERE.
-		if(istype(target, type))
-			cost = (MAX_DISSECTION_REWARD*mob_mult_list[type])
-			break
```
Comment (SomeguyManperson) [inline code/modules/surgery/experimental_dissection.dm:78]: This replaced human dissections in #2722 and human/monkey dissection is uhh bad

**Comment by Erikafox** [comment]
Yeah this absolutely needs a design talk. Feel free to pull me into one but for now I'm gonna let this ferment in my collection of dead branches.

**Pull request closed without merging**